app/templates/partials/answers/hours.html

```diff
@@ -0,0 +1,3 @@
+{% with input_type = 'hours' %}
```

Comment:
Is there a reason why there are hours.html, mins.html and timeinput.html. Don't we just need the timeinput.html?